support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -113,4 +113,25 @@ export const tests: Tests = {
 		targetPage: pageUrlRegexes.contributions.genericCheckoutOnly,
 		excludeContributionsOnlyCountries: false,
 	},
+	digitalEditionCheckout: {
+		variants: [
+			{
+				id: 'control',
+			},
+			{
+				id: 'variant',
+			},
+		],
+		audiences: {
+			ALL: {
+				offset: 0,
+				size: 1,
+			},
+		},
+		isActive: true,
+		referrerControlled: false, // ab-test name not needed to be in paramURL
+		seed: 2,
+		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
```

Comment:
Is this the correct target? It will bucket all users on the 3-tier landing page and generic checkout flows into this test. We only want to bucket users on:

- The subscribe landing page: `/subscribe`
- The digital edition checkout: `/subscribe/digitaledition`
- The generic checkout, but only when `product=DigitalSubscription`

Reply:
Thanks for the. help defining this as shown below (UK-only) ->

- targetPage: `(/uk/)(subscribe$|subscribe/digitaledition$|subscribe/digitaledition/thankyou$|checkout|thank-you)` 
- canRun will detect a new `DigitalSubscription` product from the generic `checkout` url (or undefined in the case of the old `digitalEdition` landing page)